internal/kafka/command_topic_consume.go

```diff
@@ -34,7 +34,7 @@ func (c *command) newConsumeCommand() *cobra.Command {
 			},
 			examples.Example{
 				Text: `Consume from a cloud Kafka topic named "my_topic" without logging in to Confluent Cloud.`,
-				Code: "confluent kafka topic consume my_topic --api-key <api-key> --api-secret <api-secret> --bootstrap <Kafka-cluster-endpoint> --value-format avro --schema-registry-endpoint <schema-registry-endpoint> --schema-registry-api-key <schema-registry-api-key> --schema-registry-api-secret <schema-registry-api-secret>",
+				Code: "confluent kafka topic consume my_topic --api-key my-key --api-secret my-secret --bootstrap SASL_SSL://pkc-123:9092 --value-format avro --schema-registry-endpoint https://psrc-456 --schema-registry-api-key my-sr-key --schema-registry-api-secret my-sr-secret",
```

Comment:
Nit: In reality, I think PKCs have 6 characters. How about `pkc-123456` instead?

Reply:
Also, I believe cloud API keys are 16 characters long, and only contain uppercase letters and digits

Reply:
Secrets are even longer that will make this example message really really long.. 
i can update the pkc example

Reply:
updated!